demos/python_demos/segmentation_demo/segmentation_demo.py

```diff
@@ -15,199 +15,216 @@
  limitations under the License.
 """
 
+import logging
+import random
 import sys
-import os
 from argparse import ArgumentParser, SUPPRESS
+from pathlib import Path
+from time import perf_counter
+import os
+
 import cv2
 import numpy as np
-import logging as log
 from openvino.inference_engine import IECore
 
-classes_color_map = [
-    (150, 150, 150),
-    (58, 55, 169),
-    (211, 51, 17),
-    (157, 80, 44),
-    (23, 95, 189),
-    (210, 133, 34),
-    (76, 226, 202),
-    (101, 138, 127),
-    (223, 91, 182),
-    (80, 128, 113),
-    (235, 155, 55),
-    (44, 151, 243),
-    (159, 80, 170),
-    (239, 208, 44),
-    (128, 50, 51),
-    (82, 141, 193),
-    (9, 107, 10),
-    (223, 90, 142),
-    (50, 248, 83),
-    (178, 101, 130),
-    (71, 30, 204)
+sys.path.append(str(Path(__file__).resolve().parents[1] / 'common'))
+
+from models import *
+import monitors
+from pipelines import AsyncPipeline
+from images_capture import open_images_capture
+from performance_metrics import PerformanceMetrics
+
+logging.basicConfig(format='[ %(levelname)s ] %(message)s', level=logging.INFO, stream=sys.stdout)
+log = logging.getLogger()
+
+
+CITYSCAPES_COLORS = [
```

Comment:
why cityscapes used as main color map?
we do not have any cityscapes trained models in OMZ (voc, camvid, mapillary, ade20k, camvid currently used from public) Cityscapes was used for dilation, which gone a lot of releases ago

Reply:
I took this colormap from Image Segmentation C++ version. Now I changed the default colormap according to the previous version of Python demo.

Reply:
@eaidova, do you know where did `default_color_map` come from? It would be good for the demo to name it after a dataset.
@anzhella-pankratova please add another list of colors for C++'s version matching `default_color_map`.

Reply:
I have no idea how it come from, possibly @mikhail-treskin  remember, because it come for IE python samples (git history says that he implemented it).
But as alternative, I suggest to use pascal voc colors as default, because it is the most popular among our used datasets